rir/src/api.cpp

```diff
@@ -81,14 +81,14 @@ REXPORT SEXP rir_eval(SEXP what, SEXP env) {
         f = isValidClosureSEXP(what);
     if (f == nullptr)
         Rf_error("Not rir compiled code");
-    return evalRirCode(functionCode(f), globalContext(), env, 0);
+    return evalRirCode(function2code(f), globalContext(), env, 0);
```

Comment:
not sure if I like that name, since a function has multiple code objects. so maybe bodyCode(f)?